package.json

```diff
@@ -8,11 +8,16 @@
     "web-ext": "^7.6.2"
   },
   "scripts": {
+    "build:dirty": "web-ext build -s src/",
```

Comment:
This process moved the reset step (`build:clean`) to the END of the Chrome-build script